webapp/components/self-navigator.html

```diff
@@ -138,6 +140,18 @@
         url.pathname = this.navigationPathPrefix() + testPath;
         this.navigateToLocation(url);
       }
+
+      getLocation(params) {
+        const url = new URL(location);
```

Comment:
nit: if this is `window.location` we should be consistent and qualify it as such.

Reply:
```suggestion
        const url = new URL(window.location);
```